packages/split-button/src/SplitButton/SplitButton.types.ts

```diff
@@ -0,0 +1,111 @@
+import { ComponentType, ReactFragment } from 'react';
+
+import {
+  type ButtonProps as ImportedButtonProps,
+  Variant as ButtonVariants,
+} from '@leafygreen-ui/button';
+import { DarkModeProps } from '@leafygreen-ui/lib';
+import { MenuItemProps } from '@leafygreen-ui/menu';
+import { type MenuProps as ImportedMenuProps } from '@leafygreen-ui/menu';
+import {
+  Align as ImportedAlign,
+  Justify as ImportedJustify,
+} from '@leafygreen-ui/popover';
+
+export const Variant = {
+  Default: ButtonVariants.Default,
+  Primary: ButtonVariants.Primary,
+  Danger: ButtonVariants.Danger,
+} as const;
+
+export type Variant = typeof Variant[keyof typeof Variant];
+
+type ButtonProps = Omit<
+  ImportedButtonProps,
+  'rightGlyph' | 'href' | 'as' | 'variant'
```

Comment:
did we decide not to omit href and as?

Reply:
I'm using, `InferredPolymorphic` for `SplitButton`, so i decided to omit these props from `ButtonProps` since the types are different. `<Button />` uses `Box`, so the types don't match up.

Reply:
will you leave a TODO to update this once button is using polymorphic

Reply:
and probably a ticket so we don't miss it